### PR TITLE
Style fixes 

### PR DESCRIPTION
**Pull request by fregante**
I might add here some style improvements

### CODE REVIEW

File: components/InfoPane/QueryInput.scss
Comment (fregante) [inline components/InfoPane/QueryInput.scss:1]: Is this rule now unused? Should drop it? 

https://github.com/npmgraph/npmgraph/blob/9745a479bb202324e2b50096aa05627a9b685856/components/Inspector.scss#L73

File: components/InfoPane/QueryInput.scss [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #search-field {
-  margin-left: 1em;
```
Comment (fregante) [inline components/InfoPane/QueryInput.scss:2]: ## Before

Screenshot


## After

Screenshot 1

File: components/InfoPane/QueryInput.scss
Comment (broofa) [reply components/InfoPane/QueryInput.scss:1]: It's still used.  `#tabs` is the bar at the top of the inspector.

Comment (fregante) [reply components/InfoPane/QueryInput.scss:1]: Tabs yes, but where is the input?

File: components/InfoPane/QueryInput.scss (hunk 1 above)
Comment (broofa) [reply components/InfoPane/QueryInput.scss:2]: looks good!

File: components/InfoPane/QueryInput.scss
Comment (broofa) [reply components/InfoPane/QueryInput.scss:1]: I think I see what you're referring to.  The `#tabs #search-field` in Inspector.scss rule no longer applies, so I've removed that.   Is that what you meant?

**Review by broofa — APPROVED** [review]

**Comment by broofa** [comment]
'Merged this since it'll be helpful in testing another branch I'm working on (coming soon).

**Pull request merged**